modules/showSection.js

```diff
@@ -0,0 +1,12 @@
+export default function showSection(sectionId) {
```

Comment:
- [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.